src/printer.js

```diff
@@ -111,12 +118,52 @@ function genericPrint(path, options, print) {
   ]);
 }
 
-// special layout for "chained" function calls, i.e.
-// $foo = a()
-//     ->b()
-//     ->c();
+function printPropertyLookup(path, options, print) {
```

Comment:
should these be moved inside `printMemberChain`?

Reply:
@mgrip no, we use this function inside in `printMemberChain` and output `propertylookup`